api/src/components/affiliations/controller.ts

```diff
@@ -14,27 +14,25 @@ export const updateAffiliations = async (
     try {
         const isIndependent = event.body.isIndependent;
         const affiliations = event.body.affiliations;
-        const publicationId = event.pathParameters.id;
+        const publicationVersionId = event.pathParameters.id;
         // Get publication with current version
-        const publication = await publicationService.getWithVersion(publicationId);
+        const version = await publicationVersionService.get(publicationVersionId);
```

Comment:
The comment above can be misleading after this change. It should be removed or re-phrased.

Reply:
Thanks, that's true. I just pushed a commit that rephrases this and a few similar comments elsewhere to be specific that they're dealing with versions.